backend/src/s9pk/manifest.rs

```diff
@@ -154,6 +154,29 @@ pub struct Manifest {
     pub dependencies: Dependencies,
 }
 
+impl Manifest {
+    pub fn package_procedures(&self) -> impl Iterator<Item = &PackageProcedure> {
```

Comment:
:speech_balloon: I don't like the looks of this but there are no working solutions. I think that a generator syntax would work better.